tools/js-optimizer.js

```diff
@@ -6649,22 +6649,28 @@ function emterpretify(ast) {
             if (type === ASM_INT) {
               opcode = 'ADD';
               tryNumSymmetrical();
-            } else if (type === ASM_DOUBLE) opcode = 'ADDD';
+            } else if (type === ASM_DOUBLE || type === ASM_FLOAT) {
```

Comment:
Maybe, yeah, but it would need to be something like "isTypeFloatingPoint" perhaps? To not confuse with the `float` type. But that's pretty verbose, so not sure it's worth it.